### ISSUE TITLE
Intended use for the new databricks-dolly-15k dataset

### ISSUE THREAD

**Issue by maziyarpanahi** [databrickslabs/dolly#57]
Hi,

I am really interested in the new [databricks-dolly-15k](https://github.com/databrickslabs/dolly/tree/master/data) dataset that was released today. I am just wondering, 
- Should I use it to fine-tune an already fine-tuned Dolly with `tatsu-lab/alpaca` dataset
- Or shall I just use the `databricks-dolly-15k` dataset without using `tatsu-lab/alpaca` 

I would love to see what others do with this `databricks-dolly-15k` dataset.

**Comment by srowen** [comment]
No, you would use it to fine-tune a base model to produce Dolly. You could plug it in to the existing training code that exists here, but I believe the maintainers will publish the code that trained "Dolly 2" shortly. That is already fine-tuned on this data. You could then fine-tune other models in the same way with the same data, that kind of thing.

**Comment by mateiz** [comment]
Yup, Dolly-v2 is available here: https://huggingface.co/databricks/dolly-v2-12b

**Comment by maziyarpanahi** [comment]
Many thanks @srowen and @mateiz for the help. 

Will you share the exact parameters used to fine-tune `Dolly-v2` via [train_dolly.py](https://github.com/databrickslabs/dolly/blob/master/train_dolly.py) in the README? (For reproducibility and how best to use `databricks-dolly-15k` dataset with train_dolly script)

**Comment by srowen** [comment]
Yeah I believe Matt is publishing the updated training code any moment now
